source/MaterialXTest/MaterialXFormat/XmlIo.cpp

```diff
@@ -223,6 +223,22 @@ TEST_CASE("Load content", "[xmlio]")
     mx::readFromXmlFile(parentDoc, "resources/Materials/TestSuite/libraries/metal/brass_wire_mesh.mtlx", searchPath);
     REQUIRE(parentDoc->getNodeDef("ND_TestMetal") != nullptr);
 
+    // Read a string with a relative Xinclude
```

Comment:
Some testing which was remove I think when the Syntax directory was removed a while back. Add back in searchpath usage for xincludes when parsing a document which is a string / stream (which is used for Web viewer).